### PR TITLE
Version bump to 3.4.0

### PR DESCRIPTION
**Pull request by tommyp**
This adds support for Flash messages added in #103.

### CODE REVIEW

**Comment by tijmenb** [comment]
https://github.com/alphagov/govuk_admin_template/pull/108 fixes the test failure here.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# 3.4.0
+
 * Add support for flash messages
```
Comment (tijmenb) [inline CHANGELOG.md]: I forgot to add the changelog message for https://github.com/alphagov/govuk_admin_template/pull/101

Could you add:

```
* Add optional signout element to navigation bar 
```

Comment (tommyp) [inline CHANGELOG.md]: Done.

**Comment by fofr** [comment]
Rebased against master and tests pass.

**Pull request merged**